docs/variables.md

```diff
@@ -56,7 +56,7 @@ The tag is substituted with the result of evaluating the tag as an expression.
 Note that any hyphenated parameter names or step names will cause a parsing error. You can reference them by
 indexing into the parameter or step map, e.g. `inputs.parameters['my-param']` or `steps['my-step'].outputs.result`.
 
-[Learn about the expression syntax](https://github.com/antonmedv/expr/blob/master/docs/language-definition.md).
+[Learn about the expression syntax](https://github.com/expr-lang/expr/blob/master/docs/language-definition.md).
```

Comment:
Docs on https://expr-lang.org/docs/language-definition has a better DX ;)

Reply:
Yea mentioned that in https://github.com/argoproj/argo-workflows/pull/12469#discussion_r1443145079. I was gonna make that change once this was merged, didn't want to hold up this PR for that

Reply:
Changed in #12617